### PR TITLE
chore(env-tree): display accurate warning

### PR DESCRIPTION
**Pull request by qinezh**
Since Azure account is not required for existing tab scenario, this PR is to fix the inaccurate warning under the env tree:

Before:

After:

### CODE REVIEW

File: packages/vscode-extension/src/treeview/environmentTreeItem.ts [hunk 1]
```diff
@@ -144,11 +158,18 @@ export class EnvironmentNode extends DynamicNode {
...
 class WarningNode extends DynamicNode {
-  constructor(public identifier: string, warnings: string[]) {
+  constructor(public identifier: string, accountStatus: accountStatus) {
     super(identifier, vscode.TreeItemCollapsibleState.None);
-    this.label = `Sign in with your correct Azure / M365 account`;
+    if (accountStatus.isAzureAccountLogin === false && !accountStatus.isM365AccountLogin) {
+      this.label = `Sign in with your correct Azure / M365 account`;
+    } else if (!accountStatus.isM365AccountLogin) {
+      this.label = `Sign in with your correct M365 account`;
+    } else if (!accountStatus.isAzureAccountLogin === false) {
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/treeview/environmentTreeItem.ts:167]: Should be "accountStatus.isAzureAccountLogin === false" here?

Comment (qinezh) [reply packages/vscode-extension/src/treeview/environmentTreeItem.ts:167]: nice catch!!

**Review by HuihuiWu-Microsoft — DISMISSED** [review]
LGTM

File: packages/vscode-extension/src/treeview/environmentTreeItem.ts [hunk 2]
```diff
@@ -144,11 +158,18 @@ export class EnvironmentNode extends DynamicNode {
 }
 
 class WarningNode extends DynamicNode {
-  constructor(public identifier: string, warnings: string[]) {
+  constructor(public identifier: string, accountStatus: accountStatus) {
     super(identifier, vscode.TreeItemCollapsibleState.None);
-    this.label = `Sign in with your correct Azure / M365 account`;
+    if (accountStatus.isAzureAccountLogin === false && !accountStatus.isM365AccountLogin) {
+      this.label = `Sign in with your correct Azure / M365 account`;
```
Comment (tecton) [inline packages/vscode-extension/src/treeview/environmentTreeItem.ts:164]: Do we need to localize these strings?

Comment (qinezh) [reply packages/vscode-extension/src/treeview/environmentTreeItem.ts:164]: I think so, let me move it to package.nls.json, thanks!

**Review by tecton — APPROVED** [review]

**Pull request merged**